tools/cli/utils.go

```diff
@@ -39,18 +39,16 @@ import (
 	"strings"
 	"time"
 
-	"github.com/uber/cadence/common/pagination"
-
-	"github.com/uber/cadence/client/frontend"
-	"github.com/uber/cadence/common/types"
```

Comment:
Should the imports have been reordered?

Reply:
I think it's OK.
It doesn't look correct to split them anyway.